ush/load_ufsda_modules.sh

```diff
@@ -53,6 +53,14 @@ elif [[ -d /scratch1 ]] ; then
   module load prod_util
 elif [[ -d /work ]] ; then
   # We are on MSU Orion
+  # prod_util stuff, find a better solution later...
+  #module use /apps/contrib/NCEP/hpc-stack/libs/hpc-stack/modulefiles/compiler/intel/2022.1.2/
+  #module load prod_util
+  export UTILROOT=/work2/noaa/da/python/opt/intel-2022.1.2/prod_util/1.2.2
+  export MDATE=/work2/noaa/da/python/opt/intel-2022.1.2/prod_util/1.2.2/bin/mdate
+  export NDATE=/work2/noaa/da/python/opt/intel-2022.1.2/prod_util/1.2.2/bin/ndate
+  export NHOUR=/work2/noaa/da/python/opt/intel-2022.1.2/prod_util/1.2.2/bin/nhour
+  export FSYNC=/work2/noaa/da/python/opt/intel-2022.1.2/prod_util/1.2.2/bin/fsync_file
```

Comment:
May I ask what is using this?

Reply:
Good question, there were things that were failing due to `set -e` because these weren't defined... perhaps that isn't true anymore?
It might not actually be anything besides this:
https://github.com/NOAA-EMC/global-workflow/blob/07fedaa649e4576a2bd3d2fca32bb6c144fa2a85/parm/config/config.base.emc.dyn#L148
and I just defined everyting in the prod_util module for completeness sake

Reply:
Why can't we just add prod_util to the UFSDA stack?